test/mpi/pt2pt/pssend.c

```diff
@@ -27,7 +27,7 @@ int main(int argc, char *argv[])
     }
 
     if (rank == 0) {
-        MPI_Ssend_init(NULL, 0, MPI_DATATYPE_NULL, 1, 0, MPI_COMM_WORLD, &req);
+        MPI_Ssend_init(NULL, 0, MPI_INT, 1, 0, MPI_COMM_WORLD, &req);
```

Comment:
It appears to be a ch3 bug. Did we never run into sending NULL message issues back in the ch3-days? Anyway, I made a fix here - 8c7e16ff10d586cd4a82eead10989cb131c76083

Reply:
I think its a relatively modern interpretation that you could use `MPI_DATATYPE_NULL` when count is `0`. Since it makes no difference to the test, removing it is probably best.